packages/rollup-dev-tools/test/helpers.ts

```diff
@@ -498,3 +502,56 @@ export const transpileAndDeployInitcode = async (
   const deployedViaInitcodeAddress = deployedViaInitcode.result
   return deployedViaInitcodeAddress
 }
+
+export const getMockSSTOREReplacer = (): OpcodeReplacer => {
```

Comment:
Since the result will always be the same (there are no parameters), does this need to be a function, or can it just be a class?